packages/data-transport-layer/src/services/l2-ingestion/handlers/transaction.ts

```diff
@@ -76,7 +73,34 @@ export const handleSequencerBlock = {
             r: padHexString(transaction.r, 32),
             s: padHexString(transaction.s, 32),
           }
-        ),
+        )
+      } catch (e) {
```

Comment:
Is there a specific `e` that we could catch to guarantee this is exactly the right error and not some unrelated error with `serialize`?

Reply:
^+1, my biggest confusion with this change is what would cause the error / activate this second code path, a comment to document (+ mark future deprecation?) would make reading this in the future easier!